phosphor-auditlog/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include "config.h"
+
+#include "config_main.h"
+
+#include "auditLogMgr.hpp"
+
+#include <sdbusplus/bus.hpp>
+#include <sdbusplus/server/manager.hpp>
+
+// AUDITLOG_PATH
+constexpr auto auditLogMgrRoot = "/xyz/openbmc_project/logging/auditlog";
```

Comment:
is  there a reason you aren't using the constants?
I haven't checked yet if they are being used in other places, but if they aren't you could delte them from the config.h.

Reply:
I created them to match the way other things were done. Then later I decided I could keep all of these values isolated to this subdirectory. I should have gone back and removed the #defines from the config.h file then. I'll remove them now.